CHANGES.rst

```diff
@@ -3,6 +3,8 @@
 
 - Don't show astropy-helpers version in packages that don't use it. [#16]
 
+- Remove compatibility code for Astropy < 4.0, and for astropy-helpers. [#32]
+
```

Comment:
Should I remove the other changelog entry (now obsolete) ? Or maybe we should do a release with this PR, and then another with the PR and bumping the version number to 0.2 ?

Reply:
I agree with the second option

Reply:
@pllim - See above, we were thinking about doing a release without this PR, but I'm not sure who can do releases ;)
Maybe a good idea as well to setup an Action workflow to release on PyPI.

Reply:
Oh, ops. Should I revert?

Reply:
Looks like only @astrofrog can do the release.

Reply:
Maybe a good occasion to discuss releasing with an Action workflow for infrastructure packages ? ;)
https://packaging.python.org/guides/publishing-package-distribution-releases-using-github-actions-ci-cd-workflows/

Reply:
yes that would be great!

Reply:
I thought @astrofrog and @cadair already set up Azure for that? As much as I like Actions over Azure, I think the project should be consistent across the board.

Reply:
Well the Azure solution is a bit different, it uses cibuildwheel which is needed for wheels with compiled code, but for pure Python packages it is not.  Also the pypa/gh-action-pypi-publish action is to publish directly files (sdist and wheel) to PyPI, which would simplify the release process for pytest plugins etc.

Reply:
Ah, okay. Thanks for the clarifications!

Reply:
It only uses cibuildwheel if you ask it to, it supports universal wheels and sdist without using it. In general tho I suggest we just use actions.

Reply:
I was looking at doing a release before this change, but actually #16 was merged one year ago, and the removal of astropy-helpers code started before this PR (e.g. #29, #30). So maybe we can skip this release and go straight to 0.2.0 ?

Reply:
I am okay with whatever is easier to do.